AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+|StatusCode|string|Programmatic code for the status of the assessment.|Healthy|
+|StatusMessage|string|The assessment status message.|{"code": "Healthy"}|
+|AdditionalData|string|The assessment additional data.|{"usersWithNoMfaObjectIdList": "[]"}|
+
+___
+
+### **Policy state**
+
+|Field Name|Field Type|Description|Sample value|
+|--|--|--|--|
+|JobId|int|Job ID for the scan (YYYYMMDD).|20230414|
+|PolicyDefinitionAction|string|Policy definition action, i.e. effect.|deployifnotexists|
+|PolicyAssignmentScope|string|The policy assignment scope.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|PolicyAssignmentId|string|The policy assignment ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/microsoft.authorization/policyassignments/PolicyAssignmentName|
+|PolicyDefinitionId|string|The policy assignment name.|/providers/microsoft.authorization/policydefinitions/00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|PolicyDefinitionName|string|The policy definition name.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|PolicySetDefinitionId|string|Policy set definition ID, if the policy assignment is for a policy set.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/providers/Microsoft.Authorization/policySetDefinitions/00000xxxxx000x00x0x000xx|
+|PolicySetDefinitionName|string|Policy set definition name, if the policy assignment is for a policy set.|00000xxxxx000x00x0x000xx|
+|ComplianceState|string|Compliance state of the resource.|Compliant|
+|OrgTenantId|string|The tenant ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|subscriptionid|string|The subscription ID.|00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|ResourceLocation|string|The resource location.|eastus|
+|ResourceGroup|string|The resource group name.|sampleresourcegroup|
+|ResourceType|string|The resource type.|Microsoft.Compute/virtualMachines|
+|ResourceId|string|The resource ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/resourcegroups/sampleresourcegroup/providers/microsoft.compute/virtualmachines/sampleresource|
+|StateWeight|int|The weight of policy state.|200|
+|TimeStamp|datetime|Timestamp for the policy state record.|4/10/2023 3:31:14 AM|
+|ManagementGroupIds|string|Comma-separated list of management group IDs,  which represent the hierarchy of the management groups the resource is under.|00000xxx-xx00-0x00-x0x0-00xx00000x00, 00000xxx-xx00-0x00-x0x0-00xx00000x00|
+|PolicyAssignmentParameters|string|The policy assignment parameters.|null|
+|PolicyDefinitionReferenceId|string|Reference ID for the policy definition inside the policy set, if the policy assignment is for a policy set.|null|
+|PolicyStateId|string|The policy state ID.|/subscriptions/00000xxx-xx00-0x00-x0x0-00xx00000x00/resourcegroups/sampleresourcegroup/providers/microsoft.compute/virtualmachines/sampleresource/providers/microsoft.policyinsights/policystates/00000xxxxx000x00x0x000xx00000xxxxx000x00x0x000xx00000xxxxx000x00x0x000xx|
+
+
+___
+
+### **Role definition**
```

Comment:
Role Definitions 

Have we picked up the right table name? There appears to be a miss in couple of them?

Reply:
I have refereed from one of the wiki links where we have some info. available. But agreed with your suggestion and updated according to your suggestion.